ansible/README.md

```diff
@@ -0,0 +1,22 @@
+# README
+
+This directory contains all Ansible playbooks and files to setup, configure and
+deploy services automatically.
+
+## Running Ansible playbooks
+
+You can run Ansible playbooks by executing ansible-playbook command.
+For example:
+
+`ansible-playbook --private-key=~/.ssh/id_rsa -i hosts.ini --tags=setup
+                  playbooks/jenkins/jenkins-master.yml`
```

Comment:
All YAML files in open-power-host-os/builds repo has `.yaml` extension.

Shall we keep `.yaml` extension here as well?